src/System.Runtime.InteropServices.RuntimeInformation/tests/DescriptionNameTests.cs

```diff
@@ -25,6 +26,10 @@ public void DumpRuntimeInformationToConsole()
             string lcv = PlatformDetection.LibcVersion;
 
             Console.WriteLine($@"{dvs} OS={osd} OSVer={osv} OSArch={osa} Arch={pra} Framework={frd} LibcRelease={lcr} LibcVersion={lcv}");
+
+            string tempDriveFormat = new DriveInfo(GetTestFilePath()).DriveFormat;
+            string binaryDriveFormat = new DriveInfo(typeof(object).Assembly.Location).DriveFormat;
```

Comment:
Did this work for you locally on Linux?

DriveInfo's ctor calls NormalizeDriveName.  On Windows, that uses:
https://github.com/dotnet/corefx/blob/db62f1d4cd961f74b249932b5f9bcc083e5bfa91/src/System.IO.FileSystem.DriveInfo/src/System/IO/DriveInfo.Windows.cs#L24
but on Linux there's not really an equivalent, so it just uses the full path, which won't map to any drive and will fail.  We can't use GetPathRoot on Linux, as that's just going to return '/'.